torchbeast/core/file_writer.py

```diff
@@ -37,7 +37,7 @@ def gather_metadata() -> Dict:
             is_dirty=repo.is_dirty(),
             path=repo.git_dir,
         )
-    except git.InvalidGitRepositoryError:
+    except:
```

Comment:
I'm wary of a generic except. Could you except ImportError and the exception handled before?

Reply:
I've considered this, but could not come up with a way to not nest the exceptions. I've now updated the pull request to this approach.

Reply:
Ah yes that's irritating. But still better than a plain `except:`. Thanks!